evm/test/Oracle_test.js

```diff
@@ -4,7 +4,7 @@ import { assertBigNum } from './support/matchers'
 contract('Oracle', () => {
   const sourcePath = 'Oracle.sol'
   const fHash = h.functionSelector('requestedBytes32(bytes32,bytes32)')
-  const specId = '4c7b7ffb66b344fbaa64995af81e355a'
+  const specId = h.toHex('4c7b7ffb66b344fbaa64995af81e355a')
```

Comment:
Probably better to treat this [the same way you did in the ConcreteChainlinked_test.js](https://github.com/smartcontractkit/chainlink/pull/1108/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.ts&file-filters%5B%5D=No+extension#diff-fa4dc4729365ceb66a40abe494b73a2aR17) and give it trailing zeros. We actually handle the hex-encoded hex possibility off-chain, but plan to move towards a single format in the near future.